doc/source/serve/deploy_model_tutorial.rst

```diff
@@ -0,0 +1,315 @@
+
+Since the cluster is deployed locally in this tutorial, the ``127.0.0.1:8000`` 
+refers to a localhost with port 8000. The ``[Deployment Name]`` refers to either 
+the name of the function that we called ``.deploy()`` on (in our case, this is 
+``router``), or if the ``name`` keyword parameter in ``@serve.deployment`` is 
+set, it's the value of that parameter (see the Tip under the ``router`` function 
+definition above for more info). Each ``[Parameter Name]`` refers to the name of 
+one of the fields in the request's ``query_params`` dictionary for our deployed 
+function. In our example, the only parameter we need to pass in is ``txt``. 
+This parameter is referenced in the ``txt = request.query_params["txt"]`` line 
+in the ``router`` function. Each [Parameter Name] object has a corresponding 
+[Parameter Value] object. The ``txt``'s [Parameter Value] will be a string 
+containing the article text that we want to summarize. We can chain together 
+any number of the name-value pairs using the ``&`` symbol in the request URL.
+
+Now that the ``summarize`` function is deployed on Ray Serve, we can make HTTP 
+requests to it. Here's a client script that requests a summary from the same 
+article as the original Python script:
+
+.. code-block:: python
+  import requests
+
+  article_text = ("HOUSTON -- Men have landed and walked on the moon. "        
+                                 ...
+  "of millions of people on earth.")
+  response = requests.get("http://127.0.0.1:8000/router?txt=" + article_text).text
+  print(response)
+
+  '''
+  Print output:
+
+  and walked on the moon. The astronauts of Apollo 11 steered their craft to
+  the historic landing yesterday at 4:17:40 P.M., Eastern daylight time.
+  "That's one small step for man, one giant leap for mankind," Armstrong
+  declared as he planted the first human footprint on the lunar crust. He
+  radioed to earth and the mission control room: "Houston, Tranquility Base
+  here. The Eagle has landed."
+  '''
+
+.. warning::
+
+  Since Ray Serve shuts down when the Python deployment script finishes, we can 
+  either keep Ray Serve running in the background using ``serve.start(detached=True)`` 
+  (see :doc:`deployment` for details) or for testing purposes, we can add this 
+  client script to the end of the Serve deployment script and run it all together 
+  to see the output.
+
+Our Serve deployment is still a bit inefficient though. In particular, the 
+``summarize`` function sets the input encodings and weights by defining the 
+``tokenizer`` and ``model`` variables each time that it's called. However, these 
+encodings and weights never change, so it would be better if we could define 
+``tokenizer`` and ``model`` only once and keep their values in memory instead of 
+reloading them upon each HTTP query.
+
+We can achieve this by converting our ``summarize`` function into a class:
```

Comment:
Same comment as above here: 
We probably want to add 1-2 sentences on how to run this (i.e. To deploy, first start ray by running ray start --head, then run your python script: python example.py)